src/System.Text.Json/src/System/Text/Json/Document/JsonElement.ObjectEnumerator.cs

```diff
@@ -25,34 +31,54 @@ internal ObjectEnumerator(JsonElement target)
                 _endIdx = _target._parent.GetEndIndex(_target._idx, includeEndElement: false);
             }
 
+            /// <inheritdoc />
             public JsonProperty Current =>
                 _curIdx < 0 ?
                     default :
                     new JsonProperty(new JsonElement(_target._parent, _curIdx));
 
+            /// <summary>
+            ///   Returns an enumerator that iterates the properties of an object.
+            /// </summary>
+            /// <returns>
+            ///   An <see cref="ObjectEnumerator"/> value that can be used to iterate
+            ///   through the array.
+            /// </returns>
+            /// <remarks>
+            ///   The enumerator will enumerate the properties in the order they are
+            ///   declared, and when an object has multiple definitions of a single
+            ///   property they will all individually be returned (in their correctly
+            ///   defined order).
```

Comment:
`correctly defined order`? Change to something like: `in the order in which they appear in the text`